### PR TITLE
Yogesh

### PR DESCRIPTION
**Pull request by YJB28**
I have changed it as per the Guidelines kindly review it.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 import './App.css';
 import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
+// import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// import { BrowserRouter as Router, Switch } from 'react-router-dom';
```
Comment (Techvarun1991) [inline src/App.js:6]: Remove unnecessary imports also remove spaces

File: src/Components/TEST/Data.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    {
+      project: "Project-4",
+      daily: 9,
+      weekly: 25,
+      monthly: 60,
+      quarterly: 120,
+    },
+  ];
+
+// IssueTypeBar
```
Comment (anupkore) [inline src/Components/TEST/Data.js:32]: Please Write an elaborate comment here.....Not able to understand just by IssueTypeBar

Comment (YJB28) [reply src/Components/TEST/Data.js:32]: ok I will elaborate it.

File: src/Components/AllProject.js [hunk 3]
```diff
@@ -1,21 +1,15 @@
...
-
 import { FaSortDown, FaSortUp } from "react-icons/fa";
 
 export const AllProjectList = () => {
   const maxWidth = "lg";
   const [filterStatus, setFilterStatus] = useState("All");
   const [allList , setAllList] = useState([]);
-  const [projectID , setProjectID] = useState();
 
   useEffect(() => {
```
Comment (Techvarun1991) [inline src/Components/AllProject.js:14]: Add Proper comments to the functions

File: src/Components/AllProject.js [hunk 4]
```diff
@@ -154,15 +132,15 @@ export const AllProjectList = () => {
             <div
               className="relative left-[calc(50%-11rem)] aspect-[1155/678] w-[36.125rem] -translate-x-1/2 rotate-[120deg] bg-gradient-to-r from-indigo-600 via-purple-400 to-pink-200 opacity-30 sm:left-[calc(50%-30rem)] sm:w-[72.1875rem]"
               style={{
-                // backgroundImage: `url("/Images/cool-background.svg")` ,
+
```
Comment (Techvarun1991) [inline src/Components/AllProject.js:135]: Remove spaces

File: src/Components/ProtectedRoute.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+// import React from 'react';
+// import { Redirect, Route } from 'react-router-dom';
+
+// const ProtectedRoute = ({ component: Component, ...rest }) => {
+//   const token = localStorage.getItem('token');
+//   const isAuthenticated = !!token;
```
Comment (anupkore) [inline src/Components/ProtectedRoute.js:6]: This Part is Commented...You can remove this commented part if not required in future

File: src/Components/LoginForm.js [hunk 6]
```diff
@@ -37,23 +39,30 @@ export const  LoginForm = ({ toggleSignup }) =>
...
+      AuthenticationService.SignIn(payload).then((response) => {
         const message = response.data.Return;
-        console.log(message);
-        window.location.href = '/allprojects';
+        console.log("message", message);
+        const token = response.data.token;
+        // Store the token in local storage
+        localStorage.setItem('token', token);
+        console.log("token1", token);
+        // localStorage.getItem(token);
```
Comment (Techvarun1991) [inline src/Components/LoginForm.js:50]: Remove unused commented parts

File: src/Components/LoginForm.js [hunk 7]
```diff
@@ -37,23 +39,30 @@ export const  LoginForm = ({ toggleSignup }) =>
...
-        else if(error.response.data.error === "Password is invalid")
-        {
-          setErrorMessagePassword("Invalid Password");
-        }
+        console.error('Login failed:', error);
+        // console.log(error.response.data);    
+        // console.log(error.response.data.error);
+        // if(error.response.data.error === "Email is invalid")
+        // {
+        //   setErrorMessageEmail("Invalid Email");
```
Comment (Techvarun1991) [inline src/Components/LoginForm.js:60]: Remove unused Commented parts

File: src/Components/ProtectedRoute.js (hunk 5 above)
Comment (YJB28) [reply src/Components/ProtectedRoute.js:6]: ok

Comment (Techvarun1991) [inline src/Components/ProtectedRoute.js]: Remove Comments

Comment (YJB28) [reply src/Components/ProtectedRoute.js]: I have resolved.

**Review by Techvarun1991 — CHANGES_REQUESTED** [review]
Changes Should be done as per comments

**Review by anupkore — CHANGES_REQUESTED** [review]
Suggested some changes in code...please do the needful changes

File: src/App.js (hunk 1 above)
Comment (YJB28) [reply src/App.js:6]: Ok, I will work on it.

File: src/Components/AllProject.js (hunk 3 above)
Comment (YJB28) [reply src/Components/AllProject.js:14]: Got it.

File: src/Components/ProtectedRoute.js (hunk 5 above)
Comment (YJB28) [reply src/Components/ProtectedRoute.js]: I have already removed it.

File: src/Components/LoginForm.js (hunk 6 above)
Comment (YJB28) [reply src/Components/LoginForm.js]: Sure, I will remove and commit the changes.

File: src/Components/LoginForm.js (hunk 7 above)
Comment (YJB28) [reply src/Components/LoginForm.js:59]: I have just revisited the file, the commented part is for future use. So I need that part for the authentication purpose.

File: src/Components/AllProject.js (hunk 4 above)
Comment (YJB28) [reply src/Components/AllProject.js:131]: I have resolved the Issue.

File: src/Components/ProtectedRoute.js (hunk 5 above)
Comment (anupkore) [reply src/Components/ProtectedRoute.js]: Changes Accepted

**Review by YJB28 — COMMENTED** [review]
I have made changes. Please kindly rechecked it.

File: src/Components/TEST/Data.js (hunk 2 above)
Comment (anupkore) [reply src/Components/TEST/Data.js]: Changes Accepted

File: src/Components/LoginForm.js (hunk 7 above)
Comment (Techvarun1991) [reply src/Components/LoginForm.js:59]: Ok make sure that will of use in future..

**Review by anupkore — APPROVED** [review]

**Review by Techvarun1991 — APPROVED** [review]

**Pull request merged**